runtime/lua/vim/lsp/completion.lua

```diff
@@ -424,16 +427,19 @@ end
 --- @param clients table<integer, vim.lsp.Client> # keys != client_id
 --- @param bufnr integer
 --- @param win integer
+--- @param context? lsp.CompletionContext
 --- @param callback fun(responses: table<integer, { err: lsp.ResponseError, result: vim.lsp.CompletionResult }>)
 --- @return function # Cancellation function
-local function request(clients, bufnr, win, callback)
+local function request(clients, bufnr, win, context, callback)
```

Comment:
```suggestion
local function request(clients, bufnr, win, ctx, callback)
```

Reply:
Hmm good point. For consistency with other LSP APIs I'll also change this in the public type.